templates/python/HelloWorld/gdk-config.json

```diff
@@ -4,7 +4,10 @@
       "author": "<PLACEHOLDER_AUTHOR>",
       "version": "NEXT_PATCH",
       "build": {
-        "build_system": "zip"
+        "build_system": "zip",
+        "options": {
+          "zip_name": "HelloWorld"
```

Comment:
leaving the name empty will use the component name right? That's what we want don't we?

Reply:
Artifact name in the recipe must be updated to component name as well. I'm just trying to keep it as close as to what the customers on previous versions of GDK use. I've no strong pref.

Reply:
Artifact name in recipe should be replaced automatically using `COMPONENT_NAME` right?

Reply:
That would be my ideal.

Reply:
GDK doesn't update the artifact names in the recipe (for any build system). It just updates the S3 URI path. It checks if an artifact with that name is present in the build files (`target`, `build/libs` or`zip-build`)

Reply:
I mean, a variable replacement. Don't we do a replacement for COMPONENT_NAME?

Reply:
Not really. We update `componentName`property in the recipe.